File: PRAK605/PRAK605-2210817310013-RyanMuhammadIrfan.c
#include <stdio.h>

int main()
{
    int x, i, j, k, matriksA[20][20], matriksB[20][20], hasil[20][20];

    scanf("%d", &x);

    printf("Matriks A\n");
    for (i = 0; i < x; i++)
    {
        for (j = 0; j < x; j++)
        {
            scanf("%d", &matriksA[i][j]);
        }
    }

    printf("Matriks B\n");
    for (i = 0; i < x; i++)
    {
        for (j = 0; j < x; j++)
        {
            scanf("%d", &matriksB[i][j]);
        }
    }

    for (i = 0; i < x; i++)
    {
        for (j = 0; j < x; j++)
        {
            hasil[i][j] = 0;
            for (k = 0; k < x; k++)
            {
                hasil[i][j] += matriksA[i][k] * matriksB[k][j];
            }
        }
    }

    printf("\nMatriks AXB\n");
    for (i = 0; i < x; i++)
    {
        for (j = 0; j < x; j++)
        {
            printf("%d ", hasil[i][j]);
        }
        printf("\n");
    }
}

File: PRAK605/PRAK605-2210817310013-Ryan Muhammad Irfan.py
x = int(input(""))

MatriksA = []
MatriksB = []
MatriksAB = []

print("Matriks A")
for i in range(x):
    k = list(map(int, input("").split()))
    MatriksA.append(k)

print("Matriks B")
for i in range(x):
    k = list(map(int, input("").split()))
    MatriksB.append(k)

print("Matriks AXB")
for i in range(x):
    MatriksAB.append([])
    for j in range(x):
        Hasil = 0
        for k in range(x):
            Hasil = Hasil + MatriksA[i][k] * MatriksB[k][j]
        MatriksAB[i].append(Hasil)

for number in MatriksAB:
    for k in number:
        print(k, end=" ")
    print()
